code/src/components/ErrorMessage.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+
+const backToStart = () => {
+    document.location.reload()
```

Comment:
It is an anti-pattern to reload single page application. You need to rethink the state/props structure to make your app re-render